requirements/datasets.txt

```diff
@@ -1,10 +1,10 @@
 # datasets
+faster-coco-eval==1.6.4
```

Comment:
For more stable operation, I would suggest using 1.6.6

faster-coco-eval>=1.6.6